install/installer/pkg/config/v1/config.go

```diff
@@ -143,8 +142,6 @@ type Config struct {
 
 	SSHGatewayHostKey *ObjectRef `json:"sshGatewayHostKey,omitempty"`
 
-	DisableDefinitelyGP bool `json:"disableDefinitelyGp"`
```

Comment:
Keep this, see https://github.com/gitpod-io/gitpod/pull/18492#discussion_r1291217012